cmd/catalog/cmd/generate.go

```diff
@@ -85,9 +86,33 @@ func cmdJoin(elems []string, sep string) string {
 
 // runGenerateMarkdownCmd generates a markdown test catalog.
 func runGenerateMarkdownCmd(_ *cobra.Command, _ []string) error {
-	fmt.Println("# `tnf.Test` Catalog")
+	fmt.Println("# test-network-function test case catalog")
```

Comment:
Is there a reason this section can't be replaced with a multi-line string?

Reply:
I made some heavy-handed changes to this so that static text is loaded from file rather than generated directly via Go.  The dynamic text is still generated in Go, and I would prefer to keep the multiple invocations of `fmt.Println(...)` and `fmt.Sprintf(...)` as they keep each line concise and easy to review.  The only other option I can see is to use a `Go text/template`, but that is fairly heavy-handed for something like this.